dts/bindings/i2c/espressif,esp32-i2c.yaml

```
@@ -21,23 +21,21 @@ properties:
     pinctrl-names:
       required: true
 
-    sda-pin:
-      type: int
+    scl-gpios:
+      type: phandle-array
       required: false
       description: |
-        SDA pin information is only required if
-        the target SoC does not have support in
-        hardware for clearing the I2C bus in case
-        of communication failure
+        GPIO to which the I2C SCL signal is routed. This is only required
+        if the to the target SoC does not have support in hardware for
+        clearing the I2C bus in case of communication failure
```

```suggestion
        if the target SoC does not have support in hardware for
        clearing the I2C bus in case of a communication failure
```

Same for `sda-gpios`

---

Done.